### PR TITLE
Add PhillipFreitas.md

### PR DESCRIPTION
**Pull request by phillip18**

### CODE REVIEW

File: pupilos/perfis/Phillip Freitas [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Phillip Freitas
+
+## Qual meu sonho na carreira?
+
+Atuar profissionalmente como Fullstack Web Developer, melhorar meu código e adiquirir o máximo de conhecimento possvel! 
+
+## Alguns links para me conhecer melhor
+
+[Facebook](https://www.facebook.com/phillip.jonathan.94)
```
Comment (vyper) [inline pupilos/perfis/Phillip Freitas]: Poderia adicionar seu github também, né? (:

File: pupilos/perfis/Phillip Freitas [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[Leonardo Saraiva](/mentores/perfis/leonardo_saraiva.md)
+
+## Meu Nome
+
+Phillip Freitas
+
+## Qual meu sonho na carreira?
+
+Atuar profissionalmente como Fullstack Web Developer, melhorar meu código e adiquirir o máximo de conhecimento possvel!
```
Comment (vyper) [inline pupilos/perfis/Phillip Freitas]: typo: `s/adiquirir/adquirir/`

**Comment by phillip18** [comment]
Alterado!

**Comment by phillip18** [comment]
Alterado!


Kind Regards.


*Phillip Jonathan Freitas Rosa*

Systems Analyst

+55 31 99210-8000 - Mobile
phillipjonathan@gmail.com
Linkedin:
​
https://www.linkedin.com/in/freitas-phillip-5b738a90
Github: https://github.com/phillip18

**Comment by woliveiras** [comment]
Valeu @vyper!

**Comment by woliveiras** [comment]
@phillip18, faltou somente adicionar na lista de pupilos, esse arquivo: https://github.com/training-center/mentoria/blob/master/pupilos/README.md

**Pull request merged**